migration-engine/connectors/sql-migration-connector/src/sql_schema_calculator.rs

```diff
@@ -210,7 +223,10 @@ fn calculate_relation_tables<'a>(
             ];
 
             sql::Table {
-                name: table_name,
+                name: table_name
+                    .chars()
+                    .take(configuration.max_identifier_length() - 1)
```

Comment:
I don't understand the `-1`, intuitively taking the max identifier length should result in the length being the maximum, but still ok, right?

Reply:
Good spot, that is wrong.